dashboards-observability/.cypress/integration/3_panels.spec.js

```diff
@@ -543,3 +592,31 @@ describe('Clean up all test data', () => {
   });
 });
 
+describe('Verify Operational Panels Table', () =>{
+  it('Add Dummy Data to Operational Panel and verify Table Column, Pagination and Rows Data ', () => {
+  for (let p = 0; p < 15 ; p++) { 
+  createDummyPanel(); 
```

Comment:
This line should be indented to right. Try to use some editor (like vscode or eclipse). They usually automatically indent the code inside the brackets.